ketcher-autotests/tests/Settings/Atoms/atoms-settings.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { Page, test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  TopPanelButton,
+  takeEditorScreenshot,
+  waitForPageInit,
+  pressButton,
+  drawBenzeneRing,
+  clickInTheMiddleOfTheScreen,
+  openFileAndAddToCanvas,
+  resetCurrentTool,
+  selectRing,
+  RingButton,
+  getAtomByIndex,
+} from '@utils';
+
+async function setHydrogenLabelsOn(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Atoms', { exact: true }).click();
+  const deltaX = 0;
+  const deltaY = 60;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
+  await page.getByRole('button', { name: 'Terminal and Hetero' }).click();
+  await page.getByTestId('On-option').click();
+  await pressButton(page, 'Apply');
+}
+async function selectExtendedTableElements(page: Page, element: string) {
+  await page.getByTestId('extended-table').click();
+  await page.getByRole('button', { name: element, exact: true }).click();
+  await page.getByRole('button', { name: 'Add', exact: true }).click();
+}
+async function atomDefaultSettings(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Atoms', { exact: true }).click();
+  const deltaX = 0;
+  const deltaY = 150;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
```

Comment:
I might be getting it wrong but I don't understand why do we need those manipulations in this function? 

  const deltaX = 0;
  const deltaY = 150;
  const anyX = 638;
  const anyY = 524;
  await page.mouse.move(anyX, anyY);
  await page.mouse.wheel(deltaX, deltaY);

Reply:
Scroll down the settings menu, otherwise atom menu is not visible 
![image](https://github.com/epam/ketcher/assets/82752700/92fa23b8-dc47-44c7-a051-cfc054f407f3)

Reply:
makes sense! but then "const anyX" - is not just ANY coordinate (that's why I got confused), it's very specific coordinate - like 'const scrollBarCoordinates = 638'

Reply:
maybe we could move those manipulations to a new common function with a meaningful name, then it would be more clear?

Reply:
This function is temporary as in this area there are not many locators. I think when more work is done here I can rewrite it?